SlicerSofa/SlicerSofa.py

```diff
@@ -91,15 +93,17 @@ def stopSimulation(self) -> None:
         if self._sceneUp is True:
             self._simulationRunning = False
 
+    def resetSimulation(self, parameterNode=None) -> None:
+        pass
+
```

Comment:
This function has little use as is. I would move here all the lines involving member variables of this class from https://github.com/Quilliams85/Slicer-SOFA/blob/7bbc2cea19222da2e40646b99230668d78d0bfa9/SoftTissueSimulation/SoftTissueSimulation.py#L406-L417